platforms/XboxOne/builder.js

```diff
@@ -34,4 +50,8 @@ function build(env, serverUrl) {
     const msBuildExe = path.resolve('C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/MSBuild/Current/Bin', 'MSBuild.exe');
     
     utils.spawn(msBuildExe, msBuildArgs);
+
+    const appPackagesDir = path.resolve(configDir, "AppPackages");
+    const appZipPath = path.resolve(configDir, appZipName);
```

Comment:
can you put the build into the dist/ folder instead, like we do for ps4
i.e. it is the consistent output dir for platform launcher/installer builds

Reply:
I've now put the build into the dist folder. I am unable to copy the msixbundle file on its own, so for the time being I zip the file by itself and move it into the dist folder.